testing/dart/fragment_shader_test.dart

```diff
@@ -48,33 +52,60 @@ void main() {
     expect(toFloat(renderedBytes.getUint8(3)), closeTo(1.0, epsilon));
   });
 
+  test('shader with one gradient child renders green', () async {
+    final Uint8List shaderBytes =
+        spvFile('general_shaders', 'gradient_child.spv').readAsBytesSync();
+    final FragmentShader shader = FragmentShader(
+        spirv: shaderBytes.buffer,
+        children: [_blueToGreen],
```

Comment:
how did this pass if children was not passed to the update call in the init above?